code/modules/antagonists/vampire/vampire_powers/vampire_powers.dm

```diff
@@ -31,6 +31,11 @@
 /obj/effect/proc_holder/spell/vampire/self/create_new_targeting()
 	return new /datum/spell_targeting/self
 
+/obj/effect/proc_holder/spell/vampire/after_cast(list/targets, mob/user)
```

Comment:
This override causes Glare to use 2 charges instead of 1 per cast, and doesn't seem to affect anything else (since the 33% change was reverted)